packages/ai-core/src/common/agent-service.ts

```diff
@@ -99,7 +99,7 @@ export class AgentServiceImpl implements AgentService {
     registerAgent(agent: Agent): void {
         this._agents.push(agent);
         agent.promptTemplates.forEach(
-            template => this.promptService.storePrompt(template.id, template.template)
+            template => this.promptService.storePromptTemplate(template)
```

Comment:
do we need the old method?

Reply:
This would break the API, I don't know whether people are already using it. For conveniance, I would probably leave it.

Reply:
Hmm, it is just a simpler version for all manditory values, I would keep it.